dashboards/vmagent.json

```diff
@@ -120,671 +129,672 @@
   "liveNow": false,
   "panels": [
     {
-      "collapsed": false,
+      "collapsed": true,
```

Comment:
This change collapses the panels which needs to be opened by default. See how it looks right now:
<img width="1344" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/2902918/51286450-1c62-491c-88e2-a64b14aa1cfb">